### PR TITLE
DOC: Point out that inverse modeling needs an average reference projector ready to not raise an error

### PR DESCRIPTION
**Pull request by nabilalibou**
Hello,

Some documentation enhancement discussed in #12401

### CODE REVIEW

**Review by drammock — CHANGES_REQUESTED** [review]
Text changes look good, but the rendering of the final paragraph is messed up (it seems that italicizing the whole paragraph prevents the preformatting and cross-references from being handled).  See here:

https://output.circle-artifacts.com/output/job/1e925b38-f8f2-44c7-9c40-c2c1df117da7/artifacts/0/html/auto_tutorials/preprocessing/55_setting_eeg_reference.html#eeg-reference-and-source-modeling

One fix is to just remove the `*` at beginning and end that italicizes the whole paragraph. Another option is to convert the whole paragraph into an admonition. I kinda like the admonition approach; they were designed to call attention to things, and do a better job of that than just italics. It would be something like this:

```rst
# .. important::
#     For the above reasons, when performing inverse imaging, MNE-Python will raise
#     a ``ValueError`` if there are EEG channels present and something other than
#     an average reference projector strategy has been specified. To ensure
#     correct functioning consider calling
#     :meth:`~mne.io.Raw.set_eeg_reference(projection=True)` to add an average
#     reference projector*.
```


Final note: you either need to add a changelog entry (`doc/changes/devel/12420.bugfix.rst`, see other files in that folder for format/inspiration), or add a PR label `no-changelog-entry-needed`.

File: doc/changes/devel/12401.other.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Clarify in tutorial '55_setting_eeg_reference' that an average reference projector ready is required to not raise an error during inverse modeling, by :newcontrib:`Nabil Alibou`
```
Comment (larsoner) [inline doc/changes/devel/12401.other.rst:1]:
```suggestion
Clarify in the :ref:`EEG referencing tutorial <tut-set-eeg-ref>` that an average reference projector ready is required for inverse modeling, by :newcontrib:`Nabil Alibou`
```

File: tutorials/preprocessing/55_setting_eeg_reference.py [hunk 2]
```diff
@@ -241,9 +241,12 @@
...
 #
-# For these reasons, when performing inverse imaging, *MNE-Python will raise
-# a ``ValueError`` if there are EEG channels present and something other than
-# an average reference strategy has been specified*.
+# .. important:: For these reasons, when performing inverse imaging, MNE-Python
+#                will raise a ``ValueError`` if there are EEG channels present
+#                and something other than an average reference projector strategy
+#                has been specified. To ensure correct functioning consider
+#                calling :meth:`~mne.io.Raw.set_eeg_reference` with
+#                ``(projection=True)`` to add an average reference projector.
```
Comment (larsoner) [inline tutorials/preprocessing/55_setting_eeg_reference.py:249]:
```suggestion
#                calling :meth:`set_eeg_reference(projection=True)
#                <mne.io.Raw.set_eeg_reference>` to add an average
#                reference as a projector.
```

**Review by larsoner — COMMENTED** [review]
Just two suggested tweaks to make the RST render more nicely, otherwise LGTM

**Review by drammock — APPROVED** [review]
approved pending a good-looking HTML render on CircleCI. 🤞🏻

**Comment by drammock** [comment]
oops, one more fix needed: the changelog filename should have *this PR number* (12420) not the number of the issue it fixes (12401). See https://github.com/mne-tools/mne-python/actions/runs/7803187571/job/21282378841?pr=12420#step:3:16

**Comment by larsoner** [comment]
@drammock GitHub UI allowed me to change the name, so I took care of that part!

**Pull request merged**